README.md

```diff
@@ -104,6 +104,8 @@ var server = http.createServer(function onRequest(req, res) {
 server.listen(3000);
 ```
 
+
+For sites using a more cross-browser solution this module will only provide the default, the serve-static middleware may be a better solution for any additional icons.
```

Comment:
I'm not sure what the sentence really means. Can you expand it? This module provides the _only_ cross-browser solution. It woks as far back as IE 5.5 and across all browsers and all operating systems.